tests/test_unit_conversion.py

```diff
@@ -1,7 +1,5 @@
 import os
-
```

Comment:
I don't think isort will like this!

Reply:
it's not something I changed but we'll see